serverlessworkflow/modules/ROOT/pages/use-cases/timeout-showcase-example.adoc

```diff
@@ -26,218 +33,254 @@ image::core/callback-state-timeouts.svg[]
+              "name": "printAfterEvent2",
+              "functionRef": {
+                "refName": "systemOut",
+                "arguments": {
+                  "message": "${\"event-state-timeouts: \" + $WORKFLOW.instanceId + \" executing actions for event2.\"}"
+                }
+              }
+            }
+          ]
+        }
+      ],
+      "timeouts": {
+        "eventTimeout": "PT30S"
+      },
+      "transition": "PrintExitMessage"
+    }
 ----
 
-.After executing the command, you will see an output like this:
-[source, shell]
-----
-secret/timeouts-showcase-database created
-deployment.apps/timeouts-showcase-database created
-service/timeouts-showcase-database created
-----
+== Executing the workflows
 
-[#job-service-deploy]
-== Deploying Job Service
-.Deploying Job Service
-[source, shell]
-----
-kubectl apply -f kubernetes/jobs-service-postgresql.yml
-----
+To execute the workflows you can use any of the available deployment scenarios:
 
-.After executing the command, you will see an output like this:
-[source, shell]
-----
-service/jobs-service-postgresql created
-deployment.apps/jobs-service-postgresql created
-trigger.eventing.knative.dev/jobs-service-postgresql-create-job-trigger created
-trigger.eventing.knative.dev/jobs-service-postgresql-cancel-job-trigger created
-sinkbinding.sources.knative.dev/jobs-service-postgresql-sb created
-----
+* <<execute-operator-dev-profile, Kogito Serverless Workflow Operator Dev Profile>>
 
-== Deploying the timeout showcase workflow
+* <<execute-quarkus-project-embedded-services, Quarkus Workflow Project with embedded services>>
 
-You need to build the workflow with the `knative` maven profile, then the descriptor files are generated under the `target/kubernetes` folder, and the image is pushed in the container registry.
+* <<execute-quarkus-project-standalone-services, Quarkus Workflow Project with standalone services>>
 
-.Building the timeout workflow showcase for Knative
-[source, shell]
-----
-mvn clean install -Pknative
-----
+[#execute-operator-dev-profile]
+=== Kogito Serverless Workflow Operator Dev Profile
 
-.Deploying the timeout workflow showcase in Knative
-[source, shell]
-----
-kubectl apply -f target/kubernetes/knative.yml
-kubectl apply -f target/kubernetes/kogito.yml
-----
+When you work with the {operator_name} Dev Profile, the operator will automatically provision an execution environment that contains an embedded job service instance, as well as an instance of the data index service.
+And thus, there is no need for additional configurations when you use timeouts.
 
-.After executing the commands you will see an output like this:
-[source, shell]
+To execute the workflows you must:
+
+In a command terminal, clone the `kogito-examples` repository, navigate to the cloned directory, and follow https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples/serverless-workflow-timeouts-showcase-operator-devprofile/README.md[these steps]:
```

Comment:
Also please here use the kogito-examples variable, otherwise the URL will be hard-coded to main: https://github.com/kiegroup/kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc?plain=1#L10